modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/PageToolbar.es.js

```diff
@@ -38,32 +40,38 @@ class PageToolbar extends Component {
 				aria-label={Liferay.Language.get('save')}
 				className="page-toolbar-root"
 			>
-				<ManagementToolbar.ItemList>
-					<ManagementToolbar.Item>
-						<label
-							className="toggle-switch"
-							htmlFor="active-switch-input"
-						>
-							<input
-								checked={!inactive}
-								className="toggle-switch-check"
-								id="active-switch-input"
-								onChange={onChangeActive}
-								type="checkbox"
-							/>
+				{status !== STATUS_TYPES.NOT_APPLICABLE && (
+					<ManagementToolbar.ItemList>
+						<ManagementToolbar.Item>
+							<label
+								className="toggle-switch"
+								htmlFor="active-switch-input"
+							>
+								<input
+									checked={status === STATUS_TYPES.ACTIVE}
+									className="toggle-switch-check"
+									id="active-switch-input"
+									onChange={onChangeActive}
+									type="checkbox"
+								/>
 
-							<span className="toggle-switch-bar">
-								<span className="toggle-switch-handle"></span>
-							</span>
+								<span className="toggle-switch-bar">
+									<span className="toggle-switch-handle"></span>
+								</span>
 
-							<span className="toggle-switch-text-right">
-								{inactive
-									? Liferay.Language.get('inactive')
-									: Liferay.Language.get('active')}
-							</span>
-						</label>
-					</ManagementToolbar.Item>
-				</ManagementToolbar.ItemList>
+								<span className="toggle-switch-text-right">
+									{status === STATUS_TYPES.ACTIVE
+										? Liferay.Language.get(
+												STATUS_TYPES.ACTIVE
+										  )
+										: Liferay.Language.get(
+												STATUS_TYPES.INACTIVE
+										  )}
+								</span>
```

Comment:
Oh, unfortunately the way that `Liferay.Language.get(...)` works, it doesn't take in variables. These have to be `Liferay.Language.get('inactive')` and `Liferay.Language.get('active')` in order to translate (I think because it is treated as a simple "find and replace" in the compiler).